exercise/templates/exercise/staff/results.html

```diff
@@ -10,45 +11,187 @@
+						<label for="exercise-collection">{% trans "Select exercises: " %}</label>
+					</td>
+					<td style="padding: 5px 10px 5px 10px;">
+						<select id="exercise-selection" multiple="multiple">
+						</select>
+					</td>
+				</tr>
+			</tbody>
+		</table>
+
+
+		<!--Checkboxes to select extra information about data-->
+		<div style="margin: 10px;">
+			<label style="margin-left: 10px;" class="checkbox-inline" data-toggle="tooltip"
+				title="{% trans 'Total number of submissions. Calculates all student submission counts together.' %}">
+				<input type="checkbox" class="total-subm-checkbox" value="total-subm">
+				{% trans "Total submissions" %}
+			</label>
+
+			<label style="margin-left: 10px;" class="ml-10 checkbox-inline" data-toggle="tooltip"
+				title="{% trans 'How many submissions a single student has used on the exercise on average. Only accounts for students with one or more submissions.' %}">
+				<input type="checkbox" class="avg-subm-checkbox" value="avg-subm">
+				{% trans "Average submissions per student" %}
+			</label>
+
+			<label style="margin-left: 10px;" class="ml-10 checkbox-inline" data-toggle="tooltip"
+				title="{% trans 'Maximum number of available submissions for the exercise.' %}">
+				<input type="checkbox" class="max-subm-checkbox" value="max-subm">
+				{% trans "Max submissions" %}
+			</label>
+
+			<label style="margin-left: 10px;" class="checkbox-inline" data-toggle="tooltip"
+				title="{% trans 'Number of students that have one or more exercise submissions.' %}">
+				<input type="checkbox" class="total-stu-subm-checkbox" value="total-stu-subm">
+				{% trans "Students with submissions" %}
+			</label>
+
+			<label style="margin-left: 10px;" class="checkbox-inline" data-toggle="tooltip"
+				title="{% trans 'Number of students that have received maximum points from the exercise.' %}">
+				<input type="checkbox" class="total-stu-max-checkbox" value="total-stu-max">
+				{% trans "Students with max points" %}
+			</label>
+
+			<label style="margin-left: 10px;" class="checkbox-inline" data-toggle="tooltip"
+				title="{% trans 'Average points received for the exercise per student. Only accounts for students with one or more submissions.' %}">
+				<input type="checkbox" class="avg-p-checkbox" value="avg-p">
+				{% trans "Average points per student" %}
+			</label>
+
+			<label style="margin-left: 10px;" class="checkbox-inline" data-toggle="tooltip"
+				title="{% trans 'Maximum points for the exercise.' %}">
+				<input type="checkbox" class="max-p-checkbox" value="max-p">
+				{% trans "Maximum points" %}
+			</label>
+
+			<div class="has-error">
+				<div class="checkbox">
+					<label style="margin-left: 10px;" class="checkbox-inline" data-toggle="tooltip"
+						title="{% trans 'Show unofficial points next to the official points in parentheses.' %}">
+						<input type="checkbox" class="unofficial-checkbox" value="unofficial">
+						{% trans "Show unofficial points" %}
+				  	</label>
```

Comment:
This line contains space plus tabs. The spaces are not visible, as the tab masks them out.